pkg/flink/internal/utils/output.go

```diff
@@ -14,18 +14,23 @@ func OutputErr(s string) {
 
 func OutputErrf(s string, args ...any) {
 	c := fColor.New(color.ErrorColor)
-	output.Printf(c.Sprint(s), args...)
+	output.Printf(c.Sprintln(s), args...)
 }
 
 func OutputInfo(s string) {
 	output.Println(s)
 }
 
 func OutputInfof(s string, args ...any) {
-	output.Printf(s, args...)
+	output.Printf(s+"\n", args...)
```

Comment:
I wouldn't expect any function ending in `f` to add a newline.

Reply:
removed from here and added to the caller